src/stores/FrameStore.ts

```diff
@@ -759,11 +770,24 @@ export class FrameStore {
         this.animationChannelRange = [0, frameInfo.fileInfoExtended.depth - 1];
 
         if (this.isPVImage) {
-            this.astFrameSet = this.initPVImage();
+            this.astFrameSet = this.initFrame2D();
             if (this.astFrameSet) {
                 this.spectralFrame = AST.getSpectralFrame(this.astFrameSet);
                 this.wcsInfo = AST.copy(this.astFrameSet);
             }
+        } else if (this.isUVImage) {
+            this.astFrameSet = this.initFrame();
```

Comment:
need to remind that this is not an elegant solution because there should be some redundancy between astFrameSet/astFrameSet2D, please add some comments to specify that this should be refactored in the future.

Reply:
@duidae Added a comment for this part.

Reply:
3Q